modules/node_modules/@frogpond/ccci-stolaf-college/v1/index.js

```diff
@@ -54,6 +54,7 @@ api.get('/calendar/google', calendar.google)
 api.get('/calendar/reason', calendar.reason)
 api.get('/calendar/ics', calendar.ics)
 api.get('/calendar/named/stolaf', calendar.stolaf)
+api.get('/calendar/named/stolaf-the-events-calendar', calendar.eventsCalendar)
```

Comment:
@hawkrives @rye thoughts on the name for this endpoint? Is there a shorter variant that would still provide as much clarity?

Reply:
Internally, we always called it "TEC", and I think that's what the PHP classes are prefixed with too

Reply:
I can rename this to tec!